public/data/Ashimdevnath.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Ashim Devnath",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/Ashimdevnath.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Ashimdevnath",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AshimDevnath4",
+      "icon": "twitter"
+    }
+  ],
```

Please remove this trailing comma! 

```suggestion
  ]
```

---

You can also commit these changes directly by using this `Commit suggestion` button above!